src/components/open-position.jsx

```diff
@@ -65,6 +66,23 @@ const getOpenContractsInfo = () => {
   );
 };
 
+const onClickSell = async (contract_id, indicative_price, currency) => {
+  try {
+    const resp = await sendRequest({
+      sell: contract_id,
+      price: indicative_price,
+    });
+    setBannerMessage(
+      `Contract Sold Successfully for ${currency} ${resp.sell.sold_for}`
```

Comment:
In deriv app, the message is 
![Screenshot 2023-04-12 at 11 08 33 AM](https://user-images.githubusercontent.com/84661147/231379694-485ee5ba-70e7-4f66-983b-d07b55e77f78.png)

`Contract was sold for  ${resp.sell.sold_for} ${currency} `
Do we need to be consistent with the message and currency format?

Reply:
Yes, we should be.